src/ResourceManager/Relay/Commands.Relay/Models/RelayNamespaceAttributesUpdateParameter.cs

```diff
@@ -40,7 +40,7 @@ public RelayNamespaceAttirbutesUpdateParameter(IDictionary<string, string> tags
             Tags = tags;
         }
 
-        public RelayNamespaceAttirbutesUpdateParameter(RelayNamespaceUpdateParameter relaynamespaceupdateparameter)
+        public RelayNamespaceAttirbutesUpdateParameter(RelayUpdateParameters relaynamespaceupdateparameter)
```

Comment:
the new type  has all the same properties just the name has been changed and its not exposed via cmdlet output or parameters.